README-sdk.md

```diff
@@ -0,0 +1,73 @@
+<img src="website/static/operator_logo_sdk_color.svg" height="125px"></img>
+
+[![Build Status](https://travis-ci.com/operator-framework/operator-sdk.svg?branch=master)](https://travis-ci.com/operator-framework/operator-sdk)
+[![License](http://img.shields.io/:license-apache-blue.svg)](http://www.apache.org/licenses/LICENSE-2.0.html)
+[![Go Report Card](https://goreportcard.com/badge/github.com/operator-framework/operator-sdk)](https://goreportcard.com/report/github.com/operator-framework/operator-sdk)
+
+## Documentation
+
+Docs can be found on the [Operator SDK website][sdk-docs].
```

Comment:
Should we not add a link here to open in the upstream/tag which is bumped in downstream?

Reply:
Nope, README-sdk.md is strictly a copy of upstream README.md https://github.com/openshift/ocp-release-operator-sdk/pull/102#issuecomment-779416485